django/conf/global_settings.py

```diff
@@ -587,6 +587,32 @@
     # 'django.contrib.staticfiles.finders.DefaultStorageFinder',
 ]
 
+FILE_STORAGES = {
+    'media': {
```

Comment:
`DEFAULT_FILE_STORAGE` and `MEDIA_ROOT` aren't consistent anyway. Choose whichever you prefer and explain why :-)